trie/trienode/node.go

```diff
@@ -97,6 +102,7 @@ func (set *NodeSet) AddNode(path []byte, n *Node) {
 		set.updates += 1
 	}
 	set.Nodes[string(path)] = n
+	set.mergePayload(path, n)
```

Comment:
nit: Do you need a comment like the other changes?
```suggestion
	set.mergePayload(path, n) // libevm
```

Reply:
It's the other way round, actually. The `// libevm` comments (now removed) were something I did at the _very_ beginning before we just settled on placing fields at the end of structs. The first one I ever did was an `extra *pseudo.Type` and I reflexively added the comment too while reading the original (`params`).